src/models/tiles/table/table-content.ts

```diff
@@ -338,6 +340,18 @@ export const TableContentModel = TileContentModel
       // if (self.metadata.hasExpressions) {
       //   self.metadata.updateDatasetByExpressions(self.dataSet);
       // }
+
+      if (changeType === "unlink") {
+        const title = self.title;
+        const newDataSet = createDefaultDataSet(title);
```

Comment:
This probably isn't worth worrying about right now, but I think this could create multiple datasets with the same name. If we ever exposed the datasets to users in the future (like @bacalj suggested on slack recently), this might create confusion. I guess I wouldn't recommend spending any time on it now, but I do think it's worth keeping in mind.